Mission image revamp

cimspin opened the pull request:
Fixes #261 
Closes #271 

Quite extensive changes to the way carrier mission images are handled. I think I caught everything...

gcruicks commented on MissionAlertBot.py:
```diff
@@ -974,6 +992,33 @@ async def gen_mission(ctx, carrier_name_search_term: str, commodity_search_term:
+    if os.path.isfile(f"images/{carrier_data.carrier_short_name}.png"):
+        print("Carrier mission image found, checking size...")
+        image = Image.open(f"images/{carrier_data.carrier_short_name}.png")
+        image_is_good = True if image.size == (506, 285) else False
```
You could write this as:

`image_is_good = image.size == (506, 285)`

It is implicitly True or False

cimspin replied:
Changed this and added a couple of small tweaks:
- default image option
- carrier_add ID format check (thanks BoozeBot!)
- carrier_add common info embed format on success

gcruicks reviewed (approved):
Code paths look ok, only comment is on some bool checks that could be re-written cleaner

cimspin commented:
- Choose default image randomly from selection
- Check if carrier exists before proceeding with image upload (whoops)
- Fixed a missing image_exists define
- Check to see if file is an image when user uploads an attachment
- Updated help description
